src/components/HomepageFeatures/index.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import clsx from 'clsx';
 import styles from './styles.module.css';
-import feature0 from '/img/feature-0.png';
-import feature1 from '/img/feature-1.png';
-import feature2 from '/img/feature-2.png';
-import feature3 from '/img/feature-3.png';
-import feature4 from '/img/feature-4.png';
+import feature0 from '../../../static/img/feature-0.png';
```

Comment:
Seems we can have a more concrete name for `feature0` like other images.

Reply:
I missed this one last time, how about changing it to factory.png?

Reply:
This picture is temporarily used to occupy the place, and it will be deleted after the editor is implemented.

Reply:
Got it.